tests/network/l2_bridge/utils.py

```
@@ -327,3 +336,123 @@ def create_vm_with_hot_plugged_sriov_interface(
             sriov=True,
         )
         yield vm
+
+
+def wait_for_no_packet_loss_after_connection(src_vm, dst_ip, interface=None):
+    sleep_count_value = 10
+
+    def _get_ping_state():
```

Ruff (0.14.13)

Missing return type annotation for private function `_get_ping_state`

(ANN202)